cmd/pod-scaler/frontend/src/app/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import {
+  PageSection,
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+
+const NotFound: React.FunctionComponent = () => {
+  function GoHomeBtn() {
+    const history = useHistory();
+    function handleClick() {
+      history.push('/');
+    }
+    return (
+      <Button onClick={handleClick}>Take me home</Button>
```

Comment:
Any reason not to use `Link` here? Could have the `Button` instead the `Link`.

Reply:
This was taken from the React PatternFly Seed repo verbatim, will open a PR there to update!